Tests/HttpPipelineTests/SharedMiddlewareTransformersTests.swift

```diff
@@ -68,6 +68,21 @@ class SharedMiddlewareTransformersTests: SnapshotTestCase {
     assertSnapshot(matching: middleware(conn).perform(), as: .conn)
   }
 
+  func testBasicAuth_Authorized_LowercasedHeaderName() {
```

Comment:
> @stephencelis will those tests fire automatically?

@emilwojtaszek They will on macOS! But could you run `make linux-main` from the command line root of the repo to make sure they run on Linux?

Reply:
sure